README.md

```diff
@@ -148,82 +155,83 @@ This feature is still under development, review, and testing.
-| servicePlanName`*` | `string`   |  The plan to use for the service instance |
-| servicePlanID   |  `string`   |  The plan ID in case service offering and plan name are ambiguous |
-| externalName       | `string`   |  The name for the service instance in SAP BTP, defaults to the binding `metadata.name` if not specified |
-| parameters       |  `[]object`  |  Provisioning parameters for the instance, check the documentation of the specific service you are using for details |
+| serviceOfferingName`*` | `string`  | The name of the SAP BTP service offering. |
+| servicePlanName`*` | `string`   |  The plan to use for the service instance. |
+| servicePlanID   |  `string`   |  The plan ID in case service offering and plan name are ambiguous. |
+| externalName       | `string`   |  The name for the service instance in SAP BTP, defaults to the instance `metadata.name` if not specified. |
+| parameters       |  `[]object`  |  Some services support the provisioning of additional configuration parameters during the instance creation. For the list of supported                                  parameters, check the documentation of the particular service offering.|
 
 #### Status
-| Property         | Type     | Comments                                                                                                   |
+| Property         | Type     | Description                                                                                                   |
 |:-----------------|:---------|:-----------------------------------------------------------------------------------------------------------|
-| instanceID   | `string`   | The service instance ID in SAP Cloud Service Management service |
-| operationURL | `string`   |  URL of ongoing operation for the service instance |
-| operationType   |  `string`   |  The operation type (CREATE/UPDATE/DELETE) for ongoing operation |
-| conditions       | `[]condition`   |  An array of conditions describing the status of the service instance. <br>The possible condition types are:<br>- `Ready`: set to `true`  if the instance is ready and usable<br>- `Failed`: set to `true` when an operation on the service instance fails, in this case the error details are available in the condition message.
+| instanceID   | `string`   | The service instance ID in SAP Cloud Service Management service. |
+| operationURL | `string`   |  The URL of the current operation performed on the service instance. |
+| operationType   |  `string`   |  The type of the current operation. Valid values are CREATE, UPDATE, or DELETE.  |
+| conditions       | `[]condition`   |  An array of conditions describing the status of the service instance. <br>The possible condition types are:<br>- `Ready`: set to `true`  if the instance is ready and usable<br>- `Failed`: set to `true` when an operation on the service instance fails. In this case, the details about the error are available in the        condition message.
```

Comment:
Valid values are usually used for input, as this cannot be set by users i am not sure it is the right way to describe it (same for bindings)

Reply:
Changed to "possible"